app/src/main/java/org/wikipedia/places/PlacesFragment.kt

```diff
@@ -675,6 +682,10 @@ class PlacesFragment : Fragment(), LinkPreviewDialog.LoadPageCallback, LinkPrevi
         const val CLUSTER_TEXT_LAYER_ID = "mapbox-android-cluster-text"
         const val CLUSTER_CIRCLE_LAYER_ID = "mapbox-android-cluster-circle0"
         const val ZOOM_IN_ANIMATION_DURATION = 1000
+        const val NOT_INITIALIZED = -1
```

Comment:
Naming nit: `NOT_INITIALIZED` by itself doesn't provide any context. Naming these constants something like `SURVEY_NOT_INITIALIZED`, `SURVEY_SHOW`, and `SURVEY_DO_NOT_SHOW` might provide more consistency.